src/FixedSizeArray.jl

```diff
@@ -221,7 +221,8 @@ function (::Type{<:(FixedSizeArrayBroadcastStyle{N, Mem} where {N})})(::Val{M})
     FixedSizeArrayBroadcastStyle{m, Mem}()  # `FixedSizeArray` supports any dimensionality
 end
 
-function Base.BroadcastStyle(::Type{<:(FixedSizeArray{T, N, Mem} where {T})}) where {N, Mem}
+function Base.BroadcastStyle(::Union{Type{<:(FixedSizeArray{T, N, Mem} where {T})},
+                                     Type{<:(SubArray{T, N, FixedSizeArray{T, N, Mem}} where {T})}}) where {N, Mem}
```

Comment:
> How do you mean exactly? What should be the signature and definition?

I had something like this in mind:

```suggestion
function Base.BroadcastStyle(::Type{<:(FixedSizeArray{T, N, Mem} where {T})}) where {N, Mem}
    fsa_broadcast_style(Val(N), Mem)
end

function Base.BroadcastStyle(::Type{<:(SubArray{T, N, FixedSizeArray{T, N, Mem}} where {T})}) where {N, Mem}
    fsa_broadcast_style(Val(N), Mem)
end
                                     
function fsa_broadcast_style(::Val{N}, ::Type{Mem}) where {N, Mem}
```

However I don't seem to be able to create a reproducer for this issue by trying things out in the REPL. So I guess perhaps this is not an issue at all, due to Julia's method specificity rules? Feel free to ignore the suggestion, then.